src/containers/Versions/utils.ts

```diff
@@ -42,3 +50,33 @@ export const useGetVersionValues = (cluster?: TClusterInfo, versionToColor?: Ver
 
     return versionsValues;
 };
+
+export function useVersionToColorMap(cluster?: TClusterInfo) {
+    const getVersionToColorMap = useGetClusterVersionToColorMap();
+
+    return React.useMemo(() => {
+        if (getVersionToColorMap) {
+            return getVersionToColorMap();
+        }
+        return parseVersionsToVersionToColorMap(cluster?.Versions);
+    }, [cluster?.Versions, getVersionToColorMap]);
+}
+
+/** For multi-cluster version - with using meta handlers */
+function useGetClusterVersionToColorMap(): (() => VersionToColorMap) | undefined {
+    const [clusterName] = useQueryParam('clusterName', StringParam);
+    const singleClusterMode = useTypedSelector((state) => state.singleClusterMode);
+    const {data} = clustersApi.useGetClustersListQuery(undefined, {skip: singleClusterMode});
```

Comment:
Looks like we already have versions in cluster.MapVersions - why call useGetClustersListQuery ?

Reply:
We call it to get `version_base_color_index` to make colors constant in multi-cluster version